include/picongpu/plugins/IsaacPlugin.hpp

```diff
@@ -311,7 +310,7 @@ class IsaacPlugin : public ILightweightPlugin
     {
         /* register command line parameters for your plugin */
         desc.add_options()
-            ("isaac.period", po::value< uint32_t > (&render_interval)->default_value(0),
+            ("isaac.period", po::value< uint32_t > (&render_interval),
```

Comment:
type is now `std::string` for the period but `render_interval` is still set directly here. please check :)

Reply:
I will check the behavior in a follow up after this PR